scripts/pm/hf_from_gcs.py

```diff
@@ -0,0 +1,224 @@
+from google.cloud import storage
```

Comment:
docstring for the file

What's the scope of pm?

Reply:
imho this ought to be a step (that just writes a certificate) but i'm fine with punting on that